components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/internal/IdentitySAMLSSOServiceComponent.java

```diff
@@ -78,12 +81,21 @@
     private static long defaultSingleLogoutRetryInterval = 60000;
 
     private static String ssoRedirectPage = null;
+    private static Boolean useSamlSsoResponseJspPage = false;
```

Comment:
any reason not to use primitive?

Reply:
Fixed with e0283ee8d83ebeac4d2b577a1ecd303db9be1bf0